src/org/labkey/filetransfer/view/fileTransferWebPartConfig.jsp

```diff
@@ -0,0 +1,238 @@
+<%@ taglib prefix="labkey" uri="http://www.labkey.org/taglib" %>
+<%
+    /*
+     * Copyright (c) 2017 LabKey Corporation
+     *
+     * Licensed under the Apache License, Version 2.0 (the "License");
+     * you may not use this file except in compliance with the License.
+     * You may obtain a copy of the License at
+     *
+     *     http://www.apache.org/licenses/LICENSE-2.0
+     *
+     * Unless required by applicable law or agreed to in writing, software
+     * distributed under the License is distributed on an "AS IS" BASIS,
+     * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     * See the License for the specific language governing permissions and
+     * limitations under the License.
+     */
+%>
+
+<%@ page import="org.apache.commons.lang3.StringUtils" %>
+<%@ page import="org.labkey.api.view.JspView" %>
+<%@ page import="org.labkey.api.view.Portal" %>
+<%@ page import="org.labkey.api.view.template.ClientDependencies" %>
+<%@ page import="org.labkey.filetransfer.provider.FileTransferProvider" %>
+<%@ page import="org.labkey.filetransfer.view.WebPartConfigBean" %>
+<%@ page import="java.util.Map" %>
+<%@ page extends="org.labkey.api.jsp.JspBase" %>
+<%!
+    @Override
+    public void addClientDependencies(ClientDependencies dependencies)
+    {
+        dependencies.add("sqv");
```

Comment:
Nick indicated when randomly looking over my shoulder that it wasn't necessary since sqv pulls in Ext4, but it is better to declare the dependency explicitly since we use Ext even without sqv.